ccd-definition/CaseEventToComplexTypes/draftCMO-caseManagementOrder.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
As you can see we merged Lewis changes to the CaseEventToComplexTypes folder so you can restructure those files now, if in doubt check with @lewisbirks

Reply:
Done this change